### ISSUE TITLE
"_GetCurrentProcess" and "_TransformProcessType" symbols not found - MacOS arm64, omnetpp-6.0.2

### ISSUE THREAD

**Issue by maxRN** [omnetpp/opp_env#15]
Error message appears during `opp_env install omnetpp-6.0.2` in a freshly `init`ed opp_env workspace. I'm on the latest `opp_env` version (0.22.0)

```shell
...
qrc_icons_dark.cpp
Creating shared library: /Users/maxrn/uni/omnet-test/omnetpp-6.0.2/out/clang-debug/src/qtenv/liboppqtenv_dbg.so
Undefined symbols for architecture arm64:
  "_GetCurrentProcess", referenced from:
      omnetpp::qtenv::Qtenv::doRun() in qtenv.o
  "_TransformProcessType", referenced from:
      omnetpp::qtenv::Qtenv::doRun() in qtenv.o
ld: symbol(s) not found for architecture arm64
clang-11: error: linker command failed with exit code 1 (use -v to see invocation)
make[1]: *** [Makefile:120: /Users/maxrn/uni/omnet-test/omnetpp-6.0.2/out/clang-debug/src/qtenv/liboppqtenv_dbg.so] Error 1
make: *** [Makefile:133: qtenv] Error 2
ERROR in build_omnetpp debug
ERROR The install operation stopped with error: Child process exit code 1
```

PS: Thank you for the quick work on the other issue  🎉

**Comment by rhornig** [comment]
Thanks for reporting. I'm aware of this issue. It seems that those calls were finally removed from the API. I will add some patches ASAP.

**Comment by rhornig** [comment]
As as side note. I was also able to run the IDE naively (on aarch64) after some adhoc codesigning magic. The only remaining issue is debugging as we are missing the `lldbmi2` executable, that was bundled on x86_64.

**Comment by rhornig** [comment]
please try 0.23.0

**Comment by maxRN** [comment]
Works perfectly! Thank you for this project it really makes installing Omnet so much easier.
BTW there is one more thing I noticed: 
![image](https://github.com/omnetpp/opp_env/assets/56630671/95d5cb78-5b1c-44fd-897d-20c2ebf4df57)

I get this warning every time I start a simulation, but as far as I can tell the simulations run without problems. And I know it says inet in the screenshot, but I was getting the same warning when running the tictoc or aloha examples.

**Comment by rhornig** [comment]
upps. While this issue is surfacing in the IDE, this is actually a typo in the `configure` script. It does not correctly detect the aarch64 architecture on macOS. At the end, it defaults to generate `.so` files as shared lib instead of `.dylib`. The good news is that I can patch this in `opp_env` without modifying/rebuilding the IDE. Will fix it for the next version. Thanks for reporting.

**Comment by rhornig** [comment]
ok. should be fixed now with 0.24.0. Also debugging should now work with the bundled/signed lldbmi2 tool.
